app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
**Hint** : Just like I mentioned in my second  comment, `after_save :update_comments_counter`